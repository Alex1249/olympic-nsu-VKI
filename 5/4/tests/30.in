20
a f o i r w g v u s b m k l n d z t p j
