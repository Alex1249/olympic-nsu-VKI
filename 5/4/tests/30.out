a f o i r w g v u s b m k l n j d p t z
